processor/src/main/java/org/mapstruct/ap/internal/model/source/selector/XmlElementDeclSelector.java

```diff
@@ -112,4 +119,38 @@ else if ( nameMatches.size() > 0 ) {
             return methods;
         }
     }
+
+    private Pair getMatch(SourceMethod sourceMappingMethod, final String targetPropertyName) {
```

Comment:
sometimes I make variables final because I am accessing them in an iteration. that is to show readers that these variables are to be understood as the input and everything else can be considered output of that iteration.